### PR TITLE
Support TLS for the binlog connection

### PR DESCRIPTION
**Pull request by tirsen**

### CODE REVIEW

File: go/vt/binlog/grpcbinlogplayer/player.go [hunk 1]
```diff
@@ -27,11 +27,20 @@ import (
...
 
+	"flag"
+
 	binlogdatapb "github.com/youtube/vitess/go/vt/proto/binlogdata"
 	binlogservicepb "github.com/youtube/vitess/go/vt/proto/binlogservice"
 	topodatapb "github.com/youtube/vitess/go/vt/proto/topodata"
 )
 
+var (
+	cert = flag.String("binlog_player_grpc_cert", "", "the cert to use to connect")
```
Comment (mpawliszyn) [inline go/vt/binlog/grpcbinlogplayer/player.go]: is the the binlog player cert or just the tablet cert?

Comment (tirsen) [reply go/vt/binlog/grpcbinlogplayer/player.go]: client side cert

**Comment by alainjobart** [comment]
LGTM after conflicts are resolved.

**Comment by tirsen** [comment]
@alainjobart Merge conflict resolved.

**Pull request merged**